src/implementations/DopplerLegacyClientImpl.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  DopplerLegacyClient,
+  DopplerLegacyUserData,
+} from "../abstractions/doppler-legacy-client";
+import { ResultWithoutExpectedErrors } from "../abstractions/common/result-types";
+import { AxiosResponse, AxiosStatic } from "axios";
+import { AppConfiguration } from "../abstractions";
+
+export class DopplerLegacyClientImpl implements DopplerLegacyClient {
+  private instance;
+
+  constructor({
+    axiosStatic,
+    appConfiguration: { dopplerLegacyBaseUrl },
+  }: {
+    axiosStatic: AxiosStatic;
+    appConfiguration: Partial<AppConfiguration>;
+  }) {
+    this.instance = axiosStatic.create({
+      baseURL: dopplerLegacyBaseUrl,
+      withCredentials: true,
+    });
+  }
+
+  async getDopplerUserData(): Promise<
```

Comment:
What do you think about adding a couple of tests for this function?

You can mock Axios.